test/Mono.Linker.Tests.Cases/Reflection/ActivatorCreateInstance.cs

```diff
@@ -366,6 +408,56 @@ public WithAssemblyPathParameterless (int i, object o)
 		private static void WithAssemblyPath ()
 		{
 			Activator.CreateInstanceFrom ("test", "Mono.Linker.Tests.Cases.Reflection.ActivatorCreateInstance+WithAssemblyPathParameterless");
+
+			WithNullAssemblyPath ();
+			WithEmptyAssemblyPath ();
+			WithNoValueAssemblyPath ();
+			WithAssemblyPathAndNullTypeName ();
+			WithAssemblyPathAndEmptyTypeName ();
+			WithAssemblyPathAndNoValueTypeName ();
+		}
+
+		[Kept]
+		// Technically this shouldn't warn because CreateInstanceFrom throws if the assembly file path is null.
+		// However, our implementation is the same for CreateInstance and CreateInstanceFrom.
+		[ExpectedWarning ("IL2032", nameof (Activator) + "." + nameof (Activator.CreateInstance), "assemblyFile")]
+		private static void WithNullAssemblyPath ()
+		{
+			Activator.CreateInstanceFrom (null, "Mono.Linker.Tests.Cases.Reflection.ActivatorCreateInstance+WithAssemblyPathParameterless");
```

Comment:
Not ideal, but I don't think this is that important (honestly I've never seen a call to this method in real app, we have these intrinsics for historical reasons mainly).

Reply:
We could add this to the issue - but the priority of this one is even lower.